posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+image: figures/prostate.gif
+---
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-pancreas"
+#| fig-cap: "Nine Fields of View (FOVs) of the whole-transcriptome Pancreas dataset generated with the `napari-cosmx` plugin. DAPI and PanCK is shown in blue and green, respectively. Endocrine cells in the Islets of Langerhans can be idenified by their transcript abundance (points) of marker genes. Red = _GCG_ (alpha cells), orange = _INS_ (beta cells)."
+
+knitr::include_graphics("./figures/pancreas.png")
+```
+
+# Introduction 
+
+The AtoMx Spatial Informatics Portal (SIP) is an excellent solution for analyzing 
+multi-omics spatial data generated on the CosMx Spatial Molecular Imager (SMI).
+It offers seamless integration of data analysis with data viewer in an intuitive 
+web-based interface. 
+
+For bespoke analyses, however, it can be useful to extract SMI image and transcript or 
+protein data from the SIP. Version 1.3 of the SIP introduced new exporting abilities that allow
+raw data and processed data in a variety of formats (_e.g._, Seurat, tiledb, "flat files") 
+to be downloaded. Still, how do we actually _view_ and interact with the images from these downloads?
+
+This blog post is the first in our series on using our
+`napari-cosmx` plugin. Like other items in our [CosMx Analysis Scratch Space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html),
+the `napari-cosmx` plugin is experimental so the usual [caveats]() and [license](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/license.html) applies.
+Future posts will do deep-dives into _using_ the `napari-cosmx` plugin. This post will
+show you how to:
+
+- @sec-installing Install napari and the `napari-cosmx` plugin 
+- @sec-napari-ready-files Describe what our "napari-ready" data format is 
+- @sec-loading-slides How to load a napari-ready slide 
+- @sec-stitching How to create a napari-ready slide from AtoMx raw data export 
+- @sec-adding-metadata Minimum example of how to view cell-level metadata 
+
+::: {.callout-note}
+`napari-cosmx` is actively and continuously under development in the RnD groups at
+NanoString. We do not (yet) have the source code for `napari-cosmx` opened-sourced.
+Please be aware that there may be bugs and that it has not gone through the regular
+level of quality and testing. Our goal here is to bring the capabilities of napari
+to CosMx users as fast as possible.
+:::
+
+## What is Napari?
+
+Stepping back for a second. What exacty _is_ Napari. [Napari](https://napari.org/){target="_blank"}
+is an [open-source](https://github.com/napari/napari){target="_blank"} Python project that runs a 
+Qt-based desktop GUI for interactive visualization of scientific images. 
+The application has layers of different types, similar to what you might find
+in application like Photoshop or Procreate. 
+
+## What is the `napari-cosmx` plugin?
+
+Tissue morphology layers generated by CosMx SMI are displayed as image layers 
```

Comment:
Start more general. "The napari-cosmx plugin enables viewing of data generated by the CosMx SMI platform in Napari." Or similar opening sentence.

Reply:
Updated. e8d68da48584bec9ea2c17de7b5cde7f36beb126